lnbits/docker-compose.yml

```diff
@@ -7,8 +7,7 @@ services:
       APP_PORT: $APP_LNBITS_PORT
       PROXY_AUTH_ADD: "false"
   web:
-    image: lnbitsdocker/lnbits-legend:0.8.0@sha256:3426e326bac347d09b4e018780cb1ecd8e5eac35851383d476076769ab4a9b2e
-    user: 1000:1000
```

Comment:
@louneskmt Was there some reason that this was dropped?

Reply:
They now use poetry to run the application, and it looks like it needs root permissions to be used.